stations.csv

```diff
@@ -20397,11 +20397,11 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 22383;Salve-Ruggiano;salve-ruggiano;8341062;;39.8706539;18.3113147;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8341062;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv22383;
 22384;San Lorenzo / St. Lorenzen;san-lorenzo-st-lorenzen;8302016;;46.7815190242636;11.903160810470581;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8396239;f;;f;ITKOR;t;;f;;f;;f;8396239;f;;f;8302016;t;S02016;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv22384;
 22385;San Gennaro;san-gennaro;;;41.6746;12.7264;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8302833;f;;f;;f;;f;;f;;f;8302833;f;;f;8308736;t;S08737;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv22385;
-22386;Milano Porta Garibaldi Passante;milano-porta-garibaldi-passante;;;45.484166667;9.185000000;8483;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;8389075;f;;f;;f;;f;;f;;f;;f;;f;8301662;t;;;;;f;;f;;f;;f;;f;;f;f;;Mailand;Milan;Milán;Milan;;;Milaan;Milán;;;ミラノ;밀라노;Mediolan;Milão;Милан;;;米兰;urn:trainline:public:nloc:csv22386;
+22386;Milano Porta Garibaldi Passante;milano-porta-garibaldi-passante;;;45.484166667;9.185000000;8483;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;8389075;f;;f;;f;;f;;f;;f;;f;;f;8301662;t;SO1647;N00090;;;f;;f;;f;;f;;f;;f;f;;Mailand;Milan;Milán;Milan;;;Milaan;Milán;;;ミラノ;밀라노;Mediolan;Milão;Милан;;;米兰;urn:trainline:public:nloc:csv22386;
```

Comment:
This `trenitalia_rtvt_id` is not accepted by the test. I think it is just a `O` to be replaced by a `0` in the second place.

Reply:
Addressed this problem.